Firebase/Database/Utilities/NSString+FURLUtils.m

```diff
@@ -20,7 +20,12 @@ @implementation NSString (FURLUtils)
 
 - (NSString *) urlDecoded {
     NSString* replaced = [self stringByReplacingOccurrencesOfString:@"+" withString:@" "];
+#if TARGET_OS_TV
+    NSString* decoded = [replaced stringByAddingPercentEncodingWithAllowedCharacters:
```

Comment:
Do we want to add percent encoding here? Or remove it?

Reply:
`stringByReplacingPercentEscapesUsingEncoding` has been deprecated but the replacement is `stringByRemovingPercentEncoding`.

The replacement is available on all supported platforms so I'd suggest doing that for all platforms:

https://developer.apple.com/documentation/foundation/nsstring/1409569-stringbyremovingpercentencoding?language=objc